provider/provider.go

```diff
@@ -352,7 +350,7 @@ var ObjectConfigJsonStr = `
 			"condition_value":        { "type": "string",   "optional": true, "step": "condition_details.[0].condition_value" },
 			"metric_name":            { "type": "string",   "optional": true, "step": "condition_details.[0].metric_name" },
 			"raise_for":              { "type": "command",  "optional": true, "step": "condition_details.[0].raise_for", "default": "local" },
-			"check_interval_sec":     { "type": "command",  "optional": true, "step": "check_interval" },
+			"check_interval_sec":     { "type": "command",  "optional": true, "step": "check_interval_sec", "default": 1 },
```

Comment:
What does the `step` field do? Is this causing any bugs in existing alarms?

Reply:
The "step" fields are read from step sub-objects in e.g. `get_alarm_class` instead of `list alarms`, even though they are written just like other attributes `<object>.<fileld> = <value>`.